core/controllers/diagnostic_test_player.py

```diff
@@ -43,7 +43,8 @@ class DiagnosticTestPlayerPage(
     def get(self) -> None:
         """Handles GET requests."""
         if platform_feature_services.is_feature_enabled(
-            platform_feature_list.ParamNames.DIAGNOSTIC_TEST.value):
+            platform_feature_list.ParamNames.DIAGNOSTIC_TEST.value
+        ) and constants.DEV_MODE:
```

Comment:
Wait, why is this here? I don't see why this should condition on dev mode, the is_feature_enabled() should take the server mode into account.

Reply:
I have removed it, I was doing some testing at that time.